source/includes/administration/_notification_categories.md

```diff
@@ -0,0 +1,235 @@
+## Notification Categories
+Manage notification categories.
+
+<!-------------------- LIST NOTIFICATION CATEGORIES -------------------->
+
+#### List notification categories
+
+`GET /content/categories`
+
+```shell
+# Retrieve notification categories
+curl "https://cloudmc_endpoint/v2/rest/content/categories?language=:language&organization_id=:organiationId" \
+   -H "MC-Api-Key: your_api_key"
+```
+> The above command returns JSON structured like this:
+
+```json
+{
+    "data": [
+        {
+            "organizationId": "3e824205-f507-4ffe-9891-a8addf3cf0e9",
+            "createdAt": "2020-10-29T14:06:50.000Z",
+            "translations": [
+                {
+                    "language": "en",
+                    "id": "1cea5fe9-3bfa-4eea-a9aa-8e8aa8f85346",
+                    "text": "Test-notif-reseller",
+                    "type": "title",
+                    "version": 1
+                },
+                {
+                    "language": "fr",
+                    "id": "d4044366-2bf9-4c1f-9da5-e2e47b719714",
+                    "text": "test-notif-reseller",
+                    "type": "title",
+                    "version": 1
+                }
+            ],
+            "icon": "fa fa-envelope",
+            "id": "dc921a2a-16c2-4bc3-8ed6-bcb571bae241",
+            "version": 1,
+            "updatedAt": "2020-10-29T14:06:50.000Z"
+        }
+    ]
+}
+```
+List the notification categories configured on the system.
+
+Attributes | &nbsp;
+---------- | -----------
+`id`<br/>*UUID* | The id of the notification category.
+`organizationId`<br/>*UUID* | The organization id of the notification category.
+`createdAt`<br/>*string* | The date on which the notification category was created.
+`updatedAt`<br/>*string* | The date on which the notification category was updated.
+`icon`<br/>*string* | The icon used when displaying the notification category.
+`version`<br/>*integer* | The notification category's version.
+`translations`<br/>*Array[Object]* | The translations of the notification category content.
+`translations.language`<br/>*string* | The language of that the translation is in.
+`translations.text`<br/>*string* | The content of the notification.
+`translations.type`<br/>*string* | The content that we are translating. (Must be one of: title, body, description, url slug)
```

Comment:
The content that we are translating (must be one of: title, body, description, url slug).

Reply:
40448fe47ff6582d909c3046be19ca1060e34bf7